src/app/[locale]/_component/sessionProvider.tsx

```diff
@@ -63,10 +63,10 @@ const SessionProviderComponent = ({ children }: { readonly children: React.React
   };
 
   useMemo(() => {
-    if (windowAvailable) {
+    if (windowAvailable && !EMAIL_VALIDATION_ROUTES.includes(pathName)) {
```

Comment:
As Above. What about adding an utility funciton for `EMAIL_VALIDATION_ROUTES.includes(pathName)` with a more meaningful name (ex. `weAreOnEmailValidationFlow(pathName)`, etc.). What do you think?